packages/core/src/codewhisperer/models/constants.ts

```diff
@@ -350,6 +350,10 @@ export const updateInlineLockKey = 'CODEWHISPERER_INLINE_UPDATE_LOCK_KEY'
 export const newCustomizationMessage = 'You have access to new Amazon Q customizations.'
 
 // Start of QCT Strings
+
+// feature flag for SQL transformations
+export const isSQLTransformReady = true
```

Comment:
Will update to `false`

Reply:
For future reference: feature flags should live in `dev/config.ts`, or as an "experiment" https://github.com/aws/aws-toolkit-vscode/blob/master/CONTRIBUTING.md#preview-releases-and-experiments